### PR TITLE
SASS-2042 Employment Summary Iterations

### PR DESCRIPTION
**Pull request by joeyymoore**
### Description

- Employment Summary Iterations

https://jira.tools.tax.service.gov.uk/browse/SASS-2042

##### Before Reviewing
 - [x]  Have you pulled the branch down?
 - [x]  Have you assigned yourself to the PR? 
 - [x]  Have you moved the task to “in review” on JIRA? 
 - [x]  Have you checked to ensure all dependencies are up to date? 
 - [x] Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
 - [x]  Have you run the journey tests?
 - [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x] Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
 the current pull request?
 - [x]  Have you checked to make sure there are no builds in the pipeline before you merge? 
 - [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
 - [x]  Have you run the journey tests? 
 - [x]  Have you addressed warnings where appropriate?
 - [x]  Have you rebased against the current version of main?

### CODE REVIEW

**Comment by joeyymoore** [comment]
Needs to be merged with https://github.com/hmrc/income-tax-employment-frontend/pull/268 and https://github.com/hmrc/income-tax-submission-performance-tests/pull/53

File: src/test/resources/features/99d_employment_qa_smoke.feature [hunk 1]
```diff
@@ -19,8 +19,8 @@ Feature: Smoke tests for end of year Employment Journeys to DES/IF in QA - Incom
     When the user clicks the continue button
     Then the user should be redirected to the "Your Income Tax Return" page
     When the user clicks on the employment link
-    Then the user should be redirected to the "Employment" page
-    When the user clicks on the change link
+    Then the user should be redirected to the "PAYE employment" page
+    When the user clicks on the view employment details link
```
Comment (ina-tu) [inline src/test/resources/features/99d_employment_qa_smoke.feature:23]: I think this would be the change employer selector instead

**Review by ina-tu — CHANGES_REQUESTED** [review]
needs a rebase

File: src/test/resources/features/99d_employment_qa_smoke.feature (hunk 1 above)
Comment (joeyymoore) [reply src/test/resources/features/99d_employment_qa_smoke.feature:23]: i shall double check

Comment (joeyymoore) [reply src/test/resources/features/99d_employment_qa_smoke.feature]: good spot, updated.

**Review by ina-tu — CHANGES_REQUESTED** [review]
there's a TODO in CommonStepDefs that can be removed. Everything else is fine though

**Review by ina-tu — APPROVED** [review]

**Review by ina-tu — APPROVED** [review]

**Pull request merged**